setup.cfg

```diff
@@ -7,4 +7,4 @@ testpaths = tests
 test = pytest
 [flake8]
 max-line-length = 120
-ignore = E203,W503
+ignore = E203,W503,B006
```

Comment:
FYI, B006 is the flake8-bugbear code for using mutable data structures as default parameters:
```
def bugbear_dislikes_this(sequence=[]):
    ...
```
My dev environment has flake8-bugbear enabled by default & I was tired of it complaining about this. It wouldn't be a bad idea to add bugbear to the project tools & clean up the thing it's complaining about, but I figured that was out of scope for this change.